proxy/nasshp/nassh.go

```diff
@@ -326,18 +326,26 @@ func (np *NasshProxy) allow(r *http.Request, w http.ResponseWriter, sid, hostpor
 		logid = LogId(sid, r, hostport, creds)
 	}
 	if np.filter != nil {
-		verdict := np.filter("tcp", hostport, creds)
-		if HostIp(hostport).IsUrl() {
-			for _, u := range HostIp(hostport).Resolve() {
+		host, port, err := net.SplitHostPort(hostport)
```

Comment:
Can you change my Sprintf() above (the one that creates `hostport`) to use `net.JoinHostPort`? So this is guaranteed to work correctly, even in the presence of IPv6.